navigation_speed_control:
  ros__parameters:
    controller_type: pure  # Whether to use pure/lyapunov
    field_topic: lidar_fields
    selector_switch_topic: mode_selection
    pre_navigation_topic: pre_navigation/cmd_vel
    navigation_topic: navigation/cmd_vel
    zone_info_topic: zone_info
    speed_factor: 0.8  # Factor of maximum speed when warning is active
    speed_value: 0.7   # Default velocity value if controller params unavailable
